lingpy/convert/strings.py

```diff
@@ -403,24 +405,16 @@ def multistate2nex(taxa, matrix, filename='', missing="?"):
 
 
 def write_nexus(
-        taxa, matrix, custom=None, custom_name='lingpy', missing="?", gap="-",
-        template="mrbayes.nex", filename="mrbayes.nex", dtype="RESTRICTION",
-        symbols="10",
-        commands=None, commands_name="mrbayes"):
+        wordlist, ref="cogid", custom=None,
+        custom_name='lingpy', missing="?", gap="-", template="mrbayes.nex",
+        filename="mrbayes.nex", dtype="RESTRICTION", symbols="10",
```

Comment:
Only mrbayes wants dtype=restriction (which is a MrBayes-ism only), while beast wants `standard`. This could be specified in the nexus template directly too.

Reply:
We should be able to guess at what the symbols are too.

Reply:
yes, symbols are only 10 in the current form , given that the matrix in the wordlist is evaluated and created like this. For a multi-state nexus export, we'd need some harder machinery, but this needs to be a separate function anyway.